cmd/node/config/config.toml

```diff
@@ -591,6 +575,25 @@
         # clutter the network exactly in the same moment
         MaxDeviationTimeInMilliseconds = 25
 
+[WebServerAntiflood]
+    WebServerAntifloodEnabled = true
```

Comment:
[comment-optional]

Could've simply named this one `Enabled`, since it is already part of `WebServerAntiflood`.
Also, shouldn't this have a comm?

Reply:
The name is meant to be this way, since it will make the devops process easier